.storybook/utils/styled-system-props.js

```diff
@@ -140,20 +140,299 @@ const generateStyledSystemProps = (
   ];
 };
 
+const generateStyledSystemColorProps = (
+  defaults
+) => {
+  return [
+    {
+      name: 'color',
+      type: { summary: 'string' },
+      description: 'Color, theme value or any valid CSS string.',
+      required: false,
+      defaultValue: {
+        summary: defaults.color || '-'
+      }
+    },
+    {
+      name: 'bg',
+      type: { summary: 'string' },
+      description: 'Background, theme value or any valid CSS string.',
+      required: false,
+      defaultValue: {
+        summary: defaults.bg || '-'
+      }
+    },
+    {
+      name: 'opacity',
+      type: { summary: 'decimal' },
+      description: 'Any decimal between 0 and 1.0',
+      required: false,
+      defaultValue: {
+        summary: defaults.opacity || '-'
+      }
+    }
+  ];
+};
+
+const generateStyledSystemLayoutProps = (
+  defaults
+) => {
+  return [
+    {
+      name: 'width',
+      type: { summary: 'number | string' },
+      description: 'Numbers between 0-1 are converted to a percentage, Numbers greater than 1 are converted to pixel values, Arrays are converted to responsive width styles, theme calues can be used or any valid CSS string.',
```

Comment:
```suggestion
      description: 'Numbers between 0-1 are converted to a percentage, Numbers greater than 1 are converted to pixel values, Arrays are converted to responsive width styles, theme values can be used or any valid CSS string.',
```

Reply:
Same in other descriptions

Reply:
Actually I'm not sure if this point about theme values is even appliable here, we probably won't have any theme values for widths or heights defined in theme.

So maybe something like:

Numbers between 0-1 are converted to a percentage, Numbers greater than 1 are converted to pixel values, String values are passed as raw CSS values, and Arrays are converted to responsive width styles

Reply:
Should be updated now